documentation/how-to/upgrade-server.md

```diff
@@ -48,10 +48,11 @@ Server._
 ## Appendix
 
 The table below shows a mapping between the Temporal K8s charms and the Temporal
-server versions. It can be used as a reference for upgrading your charm
-revisions in line with the Temporal server version so as to avoid any breaking
-changes.
+server versions on the **edge** channel. It can be used as a reference for
```

Comment:
out of curiosity.. isn't there a more dynamic way to show which version of charm is associated with which workload image, i feel like this info should be part of charmhub and be updated automatically